From 37686b5975fb65729e86150b46a7ceac64e3d47f Mon Sep 17 00:00:00 2001
From: Zev Weiss <zev@bewilderbeest.net>
Date: Tue, 10 May 2022 03:47:07 -0700
Subject: [PATCH 33/33] hwmon: (nct6775) Fix Kconfig references in
 SENSORS_NCT6775_I2C

In the process of renaming some Kconfig symbols during the review
process a couple of updates got omitted.  Make the i2c driver select
the correct dependency (the core module, not the platform driver), and
refer to the intended symbol for the platform driver in its help text.

Signed-off-by: Zev Weiss <zev@bewilderbeest.net>
---
 drivers/hwmon/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/hwmon/Kconfig b/drivers/hwmon/Kconfig
index 2127a907b..8c611c867 100644
--- a/drivers/hwmon/Kconfig
+++ b/drivers/hwmon/Kconfig
@@ -1471,7 +1471,7 @@ config SENSORS_NCT6775_I2C
 	tristate "I2C driver for Nuvoton NCT6775F and compatibles"
 	depends on I2C
 	select REGMAP_I2C
-	select SENSORS_NCT6775
+	select SENSORS_NCT6775_CORE
 	help
 	  If you say yes here you get support for the hardware monitoring
 	  functionality of the Nuvoton NCT6106D, NCT6775F, NCT6776F, NCT6779D,
@@ -1479,7 +1479,7 @@ config SENSORS_NCT6775_I2C
 	  Super-I/O chips via their I2C interface.
 
 	  If you're not building a kernel for a BMC, this is probably
-	  not the driver you want (see CONFIG_SENSORS_NCT6775_PLATFORM).
+	  not the driver you want (see CONFIG_SENSORS_NCT6775).
 
 	  This driver can also be built as a module. If so, the module
 	  will be called nct6775-i2c.
-- 
2.36.1.74.g277cf0bc36

